src/routes/secret.go

```diff
@@ -0,0 +1,141 @@
+package routes
+
+import (
+	"net/http"
+
+	"github.com/dana-team/platform-backend/src/controllers"
+	"github.com/dana-team/platform-backend/src/types"
+	"github.com/gin-gonic/gin"
+	"go.uber.org/zap"
+	"k8s.io/client-go/kubernetes"
+)
+
+func CreateSecret(client kubernetes.Interface, logger *zap.Logger) gin.HandlerFunc {
```

Comment:
Add a short godoc here too

Reply:
Also in all route functions, not only interfaces?

Reply:
Done.